crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -152,6 +153,42 @@ impl<'a> Commands<'a> {
         });
     }
 
+    /// Inserts a [`crate::system::System`] into a [`crate::schedule::Stage`]
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use bevy_ecs::prelude::*;
+    ///
+    /// fn another_system() {
```

Comment:
Could the example system take any param? Just to not encourage empty systems in the doc